PeerioTechnologies/peerio-mobile
# feat: e2e test for profile edit [branch ch3554]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/3554/mobile-tests-profile-settings
#### Testing instructions  
Verify that the tests work on Android. Also, if there are more ways to verify that the values / avatars were actually changed, that would be nice.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
